src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
Pls use Color instead of String. In case if u need to cast/get String -> u can do that on the point where u call this method. It's quite confusing that we call `getRandomColor` and get some `String` instead of `Color`